t/04-check_vars_docu.t

```diff
@@ -66,20 +66,17 @@ sub read_doc () {
             }
         }
     }
-    close($docfh);
 }
 
 sub write_doc () {
-    my $docfh;
-    open($docfh, '>', VARS_DOC . '.newvars');
-    print $docfh <<EO_HEADER;
+    my $data = <<EO_HEADER;
```

Comment:
Indented heredoc would make this look a bit nicer. Can we use those yet?

Reply:
Yes, we are using it in other places already